### PR TITLE
fix: fix attachments design issues

### PR DESCRIPTION
**Pull request by mikitabut**
Refs: #207

### CODE REVIEW

File: src/components/Files/FileFolderSelect.tsx [hunk 1]
```diff
@@ -141,16 +141,17 @@ export const FileFolderSelect = ({
   );
   const handleRenameFolder = useCallback(
     (newName: string, folderId: string) => {
+      const renamingFolder = folders.find((folder) => folder.id === folderId);
       const folderWithSameName = folders.find(
-        (folder) => folder.name === newName && folderId !== folder.id,
+        (folder) =>
+          folder.name === newName.trim() &&
```
Comment (IlyaBondar) [inline src/components/Files/FileFolderSelect.tsx]: Don't we still want to use case-insensitive comparison?

File: src/components/Files/PreUploadModal.tsx [hunk 2]
```diff
@@ -317,17 +318,14 @@ export const PreUploadDialog = ({
                   </div>
                   <p id={descriptionId}>
                     {t(
-                      'Max file size up to 1 GB. Supported types: {{allowedTypes}}.',
+                      'Max file size up to 512 Mb. Supported types: {{allowedTypes}}.',
```
Comment (IlyaBondar) [inline src/components/Files/PreUploadModal.tsx:321]: Do we really want to support such big files (512 Mb)?

**Review by IlyaBondar — DISMISSED** [review]

File: src/components/Files/FileFolderSelect.tsx (hunk 1 above)
Comment (mikitabut) [reply src/components/Files/FileFolderSelect.tsx]: Yes

File: src/components/Files/PreUploadModal.tsx (hunk 2 above)
Comment (mikitabut) [reply src/components/Files/PreUploadModal.tsx:321]: Yes)

**Review by IlyaBondar — APPROVED** [review]

**Pull request merged**